spring-session/src/main/java/org/springframework/session/data/gemfire/AbstractGemFireOperationsSessionRepository.java

```diff
@@ -0,0 +1,284 @@
+/*
```

Comment:
Is there good reason to have a base class right now since we only have a single repository? If not, could this be folded into a single implementation or made package private for now? This will allow us to more flexibility when updating. Of course if there is good reason for this being made available I'd like to understand why.